docs/cody/enterprise/completions-configuration.mdx

```diff
@@ -0,0 +1,186 @@
+# Completions Configuration
+
+<p className="subtitle">Learn how to configure Cody via `completions` on a Sourcegraph Enterprise instance.</p>
+
+<Callout type="info">Configuring Cody via `completions` is legacy but it's still supported. We recommend using the new [`modelConfiguration`](/cody/enterprise/model-configuration) for flexible LLM model selection.</Callout>
```

Comment:
Should we clarify here what minimum Sourcegraph version is needed for `modelConfiguration` to work, like we did above with splitting at 5.2/5.3+?

Reply:
@vdavid, we do mention the version compatibility on the main Model Configuration docs.

![CleanShot 2024-11-25 at 15 22 47@2x](https://github.com/user-attachments/assets/1ad7eabb-3f6d-44aa-9fbc-2afd6f05efbf)